AWSKinesisTests/AWSKinesisRecorderTests.m

```diff
@@ -274,6 +312,48 @@ - (void)testDiskAgeLimit {
     kinesisRecorder.diskAgeLimit = 0.0;
 }
 
+- (void)testBatchRecordsByteLimit {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Test finished running."];
+    
+    AWSKinesisRecorder *kinesisRecorder = [AWSKinesisRecorder defaultKinesisRecorder];
+    kinesisRecorder.diskByteLimit = 200 * 1024 * 1024;
```

Comment:
Please store the original value, and restore it at the bottom of the test, as noted for the test above.